examples/fluent-bit/cloudwatchlogs/README.md

```diff
@@ -2,6 +2,10 @@
 
 For documentation on Fluent Bit & CloudWatch, see: [amazon-cloudwatch-logs-for-fluent-bit](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
```

Comment:
Let's link both the C plugin docs and go plugin docs and add a short explanation here that we have two plugins with differences? You can link to: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit#new-higher-performance-core-fluent-bit-plugin